### ISSUE TITLE
ValueError: range() arg 3 must not be zero

### ISSUE THREAD

**Issue by hzxie** [Vchitect/VBench#91]
```
Running command: vbench evaluate --videos_path citydreamer4d/google-earth --dimension motion_smoothness --mode=custom_input
Found results file: ./evaluation_results/results_2024-12-29-14:18:11_eval_results.json
First value of the metric: 0.8667088050099625
Running command: vbench evaluate --videos_path citydreamer4d/google-earth --dimension dynamic_degree --mode=custom_input
Failed to find the results file in the output log: args: Namespace(output_path='./evaluation_results/', full_json_dir='/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/cli/../VBench_full_info.json', videos_path='citydreamer4d/google-earth', dimension=['dynamic_degree'], load_ckpt_from_local=None, read_frame=None, mode='custom_input', pro
mpt='None', prompt_file=None, category=None, imaging_quality_preprocessing_mode='longer')
start evaluation                                                                                                                                                              [52/1899]
Evaluation meta data saved to ./evaluation_results/results_2024-12-29-14:18:54_full_info.json
cur_full_info_path: ./evaluation_results/results_2024-12-29-14:18:54_full_info.json                                                                                                    
2024-12-29 14:18:54,525 - torch.distributed.distributed_c10d - INFO - Added key: store_based_barrier_key:1 to store for rank: 0
2024-12-29 14:18:54,526 - torch.distributed.distributed_c10d - INFO - Rank 0: Completed store-based barrier for key:store_based_barrier_key:1 with 1 nodes.
  0%|          | 0/67 [00:00<?, ?it/s]                                                                                                                                                 
  0%|          | 0/67 [00:00<?, ?it/s]                                                     
Traceback (most recent call last):                                                                                                                                                     
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/cli/../launch/evaluate.py", line 153, in <module>                                                  
    main()                                                                                                                                                                             
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/cli/../launch/evaluate.py", line 139, in main                                                      
    my_VBench.evaluate(                                                                                                                                                                
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/__init__.py", line 153, in evaluate                                                                
    results = evaluate_func(cur_full_info_path, self.device, submodules_list, **kwargs)                                                                                                
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/dynamic_degree.py", line 160, in compute_dynamic_degree                                            
    all_results, video_results = dynamic_degree(dynamic, video_list)                                                                                                                   
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/dynamic_degree.py", line 145, in dynamic_degree                                                    
    score_per_video = dynamic.infer(video_path)                    
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/dynamic_degree.py", line 67, in infer                                                              
    frames = self.get_frames(video_path)                                                                                                                                               
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/dynamic_degree.py", line 112, in get_frames                                                        
    frame_list = self.extract_frame(frame_list, interval)
  File "/mnt/slurm_home/hzxie/.conda/envs/vbench/lib/python3.9/site-packages/vbench/dynamic_degree.py", line 118, in extract_frame                                                     
    for i in range(0, len(frame_list), interval):           
ValueError: range() arg 3 must not be zero
```

**Comment by hzxie** [comment]
It seems that the `interval == 0`.

**Comment by zhangfan-p** [comment]
Thank you for pointing out this issue. The code has now been updated to address cases where the interval might be zero. The fix will be included in version 0.1.5 of the PyPI package.

**Comment by hzxie** [comment]
Fixed in f9d345a2064f90709501680c7954945149329d14.
Close #91.
